### DISCUSSION TITLE
ENTREGA 1 - PROYECTO FINAL

### DISCUSSION THREAD

**Discussion by andrea-mu** [rramosp/ai4eng.v1#146] [00 - General]
Buen día, 

- ¿Es posible que nos den ejemplos que ayuden a ilustrar los puntos (1), (2) y (3) de la primera entrega, de manera similar a los ejemplos dados del punto (4)? 

-¿Hay algún recurso o bibliografía que nos sirva para ver métricas de desempeño de Machine Learning y de Negocio que se usen actualmente y donde podamos ver para qué se usan? Esto con el fin de tener un sondeo de las métricas que podríamos usar.

Gracias

@rramosp @raulramosudea @Felipe-RA @BustamJos3

**Comment by rramosp** [comment]
hola andrea, en la página de información del curso tienes un ejemplo al
final

https://rramosp.github.io/ai4eng.v1/content/M00_intro_udea.html

On Sat, Jun 25, 2022 at 7:42 PM Andrea Muegues ***@***.***>
wrote:

> Buen día,
>
>    - ¿Es posible que nos den ejemplos que ayuden a ilustrar los puntos
>    (1), (2) y (3) de la primera entrega, de manera similar a los ejemplos
>    dados del punto (4)?
>
> -¿Hay algún recurso o bibliografía que nos sirva para ver métricas de
> desempeño de Machine Learning y de Negocio que se usen actualmente y donde
> podamos ver para qué se usan? Esto con el fin de tener un sondeo de las
> métricas que podríamos usar.
>
> Gracias
>
> @rramosp <https://github.com/rramosp> @raulramosudea
> <https://github.com/raulramosudea> @Felipe-RA
> <https://github.com/Felipe-RA> @BustamJos3 <https://github.com/BustamJos3>
>
> —
> Reply to this email directly, view it on GitHub
> <https://github.com/rramosp/ai4eng.v1/discussions/146>, or unsubscribe
> <https://github.com/notifications/unsubscribe-auth/ADJRZIIWCE7LHCPZ2MQIATTVQ6RN7ANCNFSM5Z3CYBEA>
> .
> You are receiving this because you were mentioned.Message ID:
> ***@***.***>
>

-- 
*Aviso legal:* El contenido de este mensaje y los archivos adjuntos son 
confidenciales y de uso exclusivo de la Universidad Nacional de Colombia. 
Se encuentran dirigidos sólo para el uso del destinatario al cual van 
enviados. La reproducción, lectura y/o copia se encuentran prohibidas a 
cualquier persona diferente a este y puede ser ilegal. Si usted lo ha 
recibido por error, infórmenos y elimínelo de su correo. Los Datos 
Personales serán tratados conforme a la Ley 1581 de 2012 y a nuestra 
Política de Datos Personales que podrá consultar en la página web 
www.unal.edu.co <http://www.unal.edu.co/>.* *Las opiniones, informaciones, 
conclusiones y cualquier otro tipo de dato contenido en este correo 
electrónico, no relacionados con la actividad de la Universidad Nacional de 
Colombia, se entenderá como personales y de ninguna manera son avaladas por 
la Universidad.

**Reply by BustamJos3** [reply]
Profe, se entiende el ejemplo y eso responde la 1ra parte de la duda. Pero con la segunda no hay tanta claridad. Veo que el ejemplo a penas usan 1 métrica para ML y 1 para negocio. Pero quisiéramos ver algún recurso que sea como un compendio de ese tipo de métricas y su uso usual para tener más claridad en cuanto a lo que nos podría servir más.

**Comment by rramosp** [comment]
por ejemplo:

(1) dadas las caracetrísticas de una casa (superficie, localización, etc.) vamos predecir el precio de venta de una casa en el mercado.
(2) vamos a usar el dataset de kaggle esta competición (poner el enlace), que tiene X número de muestras (casas) y tales columnas (poner la lista de columnas, o si es muy grande, poner las que se consideren más representativas para dar una idea de cómo es el dataset). 
(3) como métrica de machine learning vamos a usar el MAE (Mean Absolute Error) que es el define la propia competencia (podría ser otro si el de la competencia es complicado). Por ejemplo, si el precio de una casa es 100K y un modelo predice 120K, entonces el error es del 120%. Pero si el precio es de 40K y el modelo predice 20K el error es del 50%. Como métrica de negocio se podría usar el incremento en ventas gracias a la utilización del modelo.
(4) si las ventas no aumentan más de un 10% no merece la pena poner el modelo en producción ya que el coste de desarrollo y mantenimiento no cubriría las ganacias adicionales de ese aumento.

tened en cuenta que el proyecto va hasta la métrica de machine learning (tenéis que reportar el rendimiento de los modelos). La métrica de negocio y el punto (4) es más una reflexión de cómo se usaría el modelo es un caso hipotético de que se integrase en la operación de una empresa u organización

**Comment by juanDev-47** [comment]
buenas tardes, tengo una consulta si solo con el hecho de que todos los integrantes de mi equipo tengamos la primera entrega en el github, con eso es suficiente para dar por valida la primera entrega del proyecto o hay algun otro espacio donde debamos poner la informacion?

**Comment by rramosp** [comment]
hola juan pablo, revisa los correos que envié y la información en la página
del curso. además de subirlo en github tenéis que registrarlo en el
formulario

On Sun, Jul 3, 2022 at 6:40 PM juan pablo arenas velez <
***@***.***> wrote:

> buenas tardes, tengo una consulta si solo con el hecho de que todos los
> integrantes de mi equipo tengamos la primera entrega en el github, con eso
> es suficiente para dar por valida la primera entrega del proyecto o hay
> algun otro espacio donde debamos poner la informacion?
>
> —
> Reply to this email directly, view it on GitHub
> <https://github.com/rramosp/ai4eng.v1/discussions/146#discussioncomment-3074502>,
> or unsubscribe
> <https://github.com/notifications/unsubscribe-auth/ADJRZIKMICBRQN427VPIARTVSIQHTANCNFSM5Z3CYBEA>
> .
> You are receiving this because you were mentioned.Message ID:
> ***@***.***>
>

-- 
*Aviso legal:* El contenido de este mensaje y los archivos adjuntos son 
confidenciales y de uso exclusivo de la Universidad Nacional de Colombia. 
Se encuentran dirigidos sólo para el uso del destinatario al cual van 
enviados. La reproducción, lectura y/o copia se encuentran prohibidas a 
cualquier persona diferente a este y puede ser ilegal. Si usted lo ha 
recibido por error, infórmenos y elimínelo de su correo. Los Datos 
Personales serán tratados conforme a la Ley 1581 de 2012 y a nuestra 
Política de Datos Personales que podrá consultar en la página web 
www.unal.edu.co <http://www.unal.edu.co/>.* *Las opiniones, informaciones, 
conclusiones y cualquier otro tipo de dato contenido en este correo 
electrónico, no relacionados con la actividad de la Universidad Nacional de 
Colombia, se entenderá como personales y de ninguna manera son avaladas por 
la Universidad.
